### PR TITLE
murphey's law

### PR DESCRIPTION
**Pull request by lawnsea**
- [x] @aaronj1335, i don't know how to preview this thingie. please make sure it's not busted
- [x] @aaronj1335, we need to get sponsor approval of the logo, etc. 
- [ ] @rmurphey, are you happy with the talk synopsis as well as your bio and branding™

### CODE REVIEW

File: _posts/2016-06-21-meetup.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+layout: post
+categories:
+- posts
+- meetups
+when: 2016-06-21T19:30:00-05:00
+---
+
+{% assign speakr = 'Rebecca Murphey' %}
+{% assign handle = 'rmurphey' %}
+{% assign twiturl = 'http://twitter.com/rmurphey' %}
```
Comment (rmurphey) [inline _posts/2016-06-21-meetup.md]: i hear all the cool kids use https these days

File: _posts/2016-06-21-meetup.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    Rebecca is a software engineer at Indeed, where she leads efforts to
+    evangelize, standardize, and implement front-end best practices across the
+    applications that power the world’s number one job search site.
+
+    or something
+  </div>
+</div>
+
+[Frog][] is hosting us this month. You folks are awesome as always. We'll also be enjoying food and beverages this time thanks to [CloudFlare][sponsor].
```
Comment (rmurphey) [inline _posts/2016-06-21-meetup.md]: I think you left out a Frog.

**Comment by rmurphey** [comment]
a++ would speak again.

File: _posts/2016-06-21-meetup.md (hunk 2 above)
Comment (aaronj1335) [inline _posts/2016-06-21-meetup.md]: it's cool, the markdown parser they use will figure it out.

**Comment by aaronj1335** [comment]
lawnsea/austinjavascript.github.io#4

i reached out to dane about the logo, but let's not let that hold up posting.

**Pull request closed without merging**